test/kubernetes_test.go

```diff
@@ -415,18 +504,355 @@ var dnsTestCasesAllNSExposed = []test.Case{
 
 var dnsTestCasesFallthrough = []test.Case{
 	{
+		Qname: "svc-1-a.test-1.svc.cluster.local.", Qtype: dns.TypeA,
```

Comment:
The added test case are not new test cases, they are test case that have two different answers based on the coredns configuration.